dl/src/main/scala/com/intel/analytics/bigdl/optim/SGD.scala

```diff
@@ -146,21 +171,21 @@ object SGD {
     }
   }
 
-  case class EpochDecay(decayType: (Int) => Double) extends LearningRateSchedule {
+  case class EpochDecay(decayType: (Int) => Double) extends HyperParameterScheduler {
     override def updateHyperParameter(config: Table, state: Table): Unit = {
       val lr = config.get[Double]("learningRate").getOrElse(1e-1)
-      var clr = -lr
+      var clr = lr
```

Comment:
why change this? If you want to display a positive learningRate, you should negative it at the display part.